src/internal/tester/config.go

```diff
@@ -120,6 +122,12 @@ func readTestConfig() (map[string]string, error) {
 		os.Getenv(EnvCorsoSecondaryM365TestUserID),
 		vpr.GetString(TestCfgSecondaryUserID),
 		"AdeleV@10rqc2.onmicrosoft.com")
+	fallbackTo(
+		testEnv,
+		TestCfgTertiaryUserID,
+		os.Getenv(EnvCorsoTertiaryM365TestUserID),
+		vpr.GetString(TestCfgTertiaryUserID),
+		"PradeepG@10rqc2.onmicrosoft.com")
```

Comment:
I've added a new user to make sure that we have enough permissions combinations to test all the different scenarios. @ashmrtn will need you help to add another CI variable for this.